rosette/rbl/rosette/namespace-util.rbl

```diff
@@ -0,0 +1,119 @@
+;;; TODO: Remove unused methods in Tuplespace and add tests for the rest
+
+;;; TODO: Add tag for whether persistent
+;;; TODO: Translate match in compiler
+
+(random-number-init)
+;;; The debugSymbol is inserted so that the output is easier to read
+(defOprn generateFresh)
+(defProc (generateFresh debugSymbol) (->symbol (string-concat "Rholang-" debugSymbol "-" (->string (random-number)))))
```

Comment:
This is called `gensym` in Common Lisp, but usually it's a monotonically increasing number on the backend:
```Lisp
CL-USER> (loop for i below 10 collect (gensym "Rholang-"))
(#:|Rholang-535| #:|Rholang-536| #:|Rholang-537| #:|Rholang-538|
 #:|Rholang-539| #:|Rholang-540| #:|Rholang-541| #:|Rholang-542|
 #:|Rholang-543| #:|Rholang-544|)
```
That random number may play havok over time.

Reply:
I've added this as https://rchain.atlassian.net/browse/ROS-339 as I don't want to touch C++ in this PR.